packages/mesos/extra/start_mesos.sh

```diff
@@ -27,7 +27,7 @@ if [[ "${distro}" == 'coreos' ]]; then
        coreos_networkd_config
        echo "Configuring systemd-networkd to ignore docker bridge and DC/OS overlay interfaces..."
 
-       if systemctl is-enabled systemd-networkd > /dev/null; then
+       if systemctl is-active systemd-networkd > /dev/null; then
```

Comment:
Why is this change?

Reply:
because the unit is disabled and active in the AMIs published by coreOS

Reply:
Looks like we only care about the exit code in the condition. but we want the networkd to be **active**

* enabled - a service (unit) is configured to start when the system boots
* active - a service (unit) is currently running.

https://www.digitalocean.com/community/tutorials/how-to-use-systemctl-to-manage-systemd-services-and-units

```
There are also methods for checking for specific states. For instance, to check to see if a unit is currently active (running), you can use the is-active command:

systemctl is-active application.service
This will return the current unit state, which is usually active or inactive. The exit code will be "0" if it is active, making the result simpler to parse programmatically.
To see if the unit is enabled, you can use the is-enabled command:

systemctl is-enabled application.service
This will output whether the service is enabled or disabled and will again set the exit code to "0" or "1" depending on the answer to the command question.
```